src/lean_dojo/data_extraction/lean.py

```diff
@@ -43,27 +46,46 @@
     )
     GITHUB = Github()
 
-LEAN4_REPO = GITHUB.get_repo("leanprover/lean4")
+# LEAN4_REPO = GITHUB.get_repo("leanprover/lean4")
+LEAN4_REPO = None
```

Comment:
This is the best part for me as a LeanDojo user! I sometimes hit the GitHub rate limits just by importing `lean_dojo` because of these lines. I much prefer a fast, side-effect-free import! Thanks!